src/PrestaShopBundle/Resources/views/Admin/Login/Blocks/login_form.html.twig

```diff
@@ -0,0 +1,102 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{{ form_start(loginForm, { action: path('_admin_login') }) }}
+
+{% set inputIds = {
```

Comment:
Why use this ? It does not seem to help so much making the file readable 🤔you did not want to use hardcoded ID ?

Reply:
I wanted to keep all IDs in one place. Since they are inherited from legacy and might be changed in the far future, I thought it's better to not "spread" them all over :slightly_smiling_face:

Reply:
If you want to keep this IDs in one place it would be better to have them in the `FormType` as const rather than in a twig templates This will make them more easily reusable in other places and it will make this template cleaner

Reply:
@jolelievre Err ... it would be mixing some display matters with the form logic 😅